examples/web/src/profiles/UseSetProfileMetadata.tsx

```diff
@@ -24,6 +24,11 @@ function UpdateProfileForm({ activeProfile }: UpdateProfileFormProps) {
     const location = formData.get('location') as string | undefined;
     const website = formData.get('website') as string | undefined;
 
+    if (!name || !bio || !location || !website) {
+      toast.error('Please fill all fields');
+      return;
+    }
```

Comment:
This cannot really happen with HTML5 validation in place.

Do we need to be this strict? I relaxed this example cause it was a little annoying to use.

Reply:
It's very unlikely but people can manipulate the HTML to remove the requires attrs. 

This was more for type safety and avoiding adding an empty location/website attr, that seems undesirable